docs/python/debugging.md

```diff
@@ -383,7 +391,13 @@ Specifies how program output is displayed as long as the defaults for `redirectO
 |----------------------------------|--------------------------------------------------------------------|
 | `"internalConsole"`              | **VS Code debug console.** If `redirectOutput` is set to False, no output is displayed.                                 |
 | `"integratedTerminal"` (default) | [VS Code Integrated Terminal](/docs/editor/integrated-terminal.md). If `redirectOutput` is set to True, output is also displayed in the debug console.|
-| `"externalTerminal"`             | **Separate console window**. If `redirectOutput` is set to True, output is also displayed in the debug console.                                            |
+| `"externalTerminal"`             | **Separate console window**. If `redirectOutput` is set to True, output is also displayed in the debug console. |
+
+### `purpose`
+ If set to `debug-test` defines that the configuration should be used when debugging tests in VS Code.
+ If set to `debug-in-terminal`, defines that the configuration should be used when and only when using the  **Debug Python File in Terminal** button on the top-right of the editor.
+ Note that the `purpose` option can't be used to start the debugger through F5 or Run > Start Debugging.
```

Comment:
Does F5 need to be in kbstyle or anything?

Reply:
great catch!!